Content.Client/Options/UI/OptionsMenu.xaml.cs

```diff
@@ -16,6 +21,13 @@ public OptionsMenu()
             Tabs.SetTabTitle(1, Loc.GetString("ui-options-tab-controls"));
             Tabs.SetTabTitle(2, Loc.GetString("ui-options-tab-audio"));
             Tabs.SetTabTitle(3, Loc.GetString("ui-options-tab-network"));
+
+            UpdateTabs();
+        }
+
+        public void UpdateTabs()
+        {
+            GraphicsTabControl?.UpdateProperties();
```

Comment:
Is this even necessary? UpdateTabs is only ever called during window startup, not on cvar change, so this seemingly does absolutely nothing except repeat work that the config tabs do already.

Reply:
It is called when the window is opened not just the startup. Please see [Content.Client/UserInterface/Systems/EscapeMenu/OptionsUIController.cs:50](https://github.com/space-wizards/space-station-14/pull/20272/files#diff-bfed2ddc0227d98d1743d18c7837d472dd7995c054ef1ebc25109fe4c38a4945R50)